File: 1/1.js
/*
 * File: 1.js
 * Description: http://projecteuler.net/problem=1
 */

var sum = 0;
for(i = 3; i < 1000; i++) 
    if (!(i % 3) || !(i % 5))
        sum += i;
print(sum)

File: 1/1.c
/*
 * File: 1.c
 * Description: http://projecteuler.net/problem=1
 */

#include<stdio.h>

void main() {
    int i, sum = 0;
    for(i = 3; i < 1000; i++)
        if (!((i % 5) && (i % 3)))
            sum += i;
    printf("%d\n", sum);
}
